src/get-options-overrides.ts

```diff
@@ -0,0 +1,11 @@
+import {CompilerOptions, ModuleKind} from "typescript";
+
+export function getOptionsOverrides(): CompilerOptions {
+	return {
+		module: ModuleKind.ES2015,
+		noEmitHelpers: true,
+		importHelpers: true,
+		noResolve: false,
+		outDir: process.cwd(),
```

Comment:
Nice catch in terms of declarationDir from tsconfig :-)

My expectation is for Rollup to be the decider of the placement of declaration files. The tsconfig properties that relate to output directories or filenames, in my opinion, should be overridden so that the destination that the consumers of the plugin defines in Rollup's configuration is always used. That is also the only way to make Rollup's multiple target build-step with declarations "just work" without having to maintain multiple tsconfig files with different declaration directories.